demo/openssl-qat-engine/Dockerfile

```diff
@@ -1,89 +1,4 @@
-FROM ubuntu:22.04 as builder
-
-ARG QATLIB_VERSION="24.02.0"
-ARG QAT_ENGINE_VERSION="v1.5.0"
-ARG ASYNC_NGINX_VERSION="v0.5.1"
-ARG IPSEC_MB_VERSION="v1.5"
-ARG IPP_CRYPTO_VERSION="ippcp_2021.11.0"
+FROM ubuntu:24.04 as builder
```

Comment:
```suggestion
FROM ubuntu:24.04
```

Reply:
Ah careless mistake. Let me fix.

Reply:
no problem. this commit could go to its own PR also to make this PR a bit smaller